### PR TITLE
OSDOCS 7182: Removing unsupported OSSO versions and adding note about…

### PR DESCRIPTION
**Pull request by bergerhoffer**
… being released in the future

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-7182

Link to docs preview:
https://62979--docspreview.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html

QE review:
- [x] QE has approved this change.

Additional information:
Note that "Branch Build" will render as "4.14" once the PR merges

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@ingvagabund @kasturinarra This PR removes the OSSO versions 1.1.0 and 1.1.1 since they aren't supported with 4.14.

Also, since only OSSO 1.2.0 will be supported with 4.14, and that it won't be released until some time after 4.14 GA, I'm adding a temporary note in that gap where no OSSO version is available. Once OSSO 1.2.0 is released, I will remove this note and replace it with the 1.2.0 release notes.

Let me know if this plan sounds okay!

Preview: https://62979--docspreview.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html

File: snippets/operator-not-available.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// When including this file, ensure that {operator-name} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[IMPORTANT]
+====
+{operator-name} is not available yet for {product-title} {product-version}. The Operator is planned to be released in the near future.
```
Comment (mburke5678) [inline snippets/operator-not-available.adoc:6]: Tiny nit. _not available yet for_ sounds a bit  awkward. Maybe _not yet available for_ or _not currently available for_ ??

**Comment by mburke5678** [comment]
@bergerhoffer One tiny nit. Feel free to ignore, as this seems to be a short-lived addition to the docs.

**Comment by kasturinarra** [comment]
@bergerhoffer currently the release date for SSO 1.2.0 is set as 1st novemeber 2023 just after the 4.14 GA which is on 31st october, 2023. So i am not sure if we really need to add this in the release note as the time difference is like just a day. Though i am not sure how it would be handled from docs point of view, if this is the way to go i am fine with the release notes, but one comment here is instead of `operator planned to released in the near future` we should say something else as `near future` could mean a 6 months to 1 year may be ?` WDYT ? Also please help correct me if my understanding is not right.

File: snippets/operator-not-available.adoc (hunk 1 above)
Comment (bergerhoffer) [reply snippets/operator-not-available.adoc:6]: Good suggestion, I went with "not currently available"

**Comment by bergerhoffer** [comment]
Thanks, I'll go ahead and merge this in then, and if we need to make any adjustments, we can do that in a separate PR. Thanks!

**Pull request merged**